dataset-workflows/simple_s3_ingest/cdk/cdk_stack.py

```diff
@@ -0,0 +1,180 @@
+import os
+import config
+from aws_cdk import core, aws_iam, custom_resources
+import aws_cdk.aws_stepfunctions as stepfunctions
+import aws_cdk.aws_events as events
+import aws_cdk.aws_events_targets as targets
+from aws_cdk import aws_secretsmanager as secretsmanager
+from aws_cdk import aws_lambda
+from aws_cdk import aws_stepfunctions_tasks as tasks
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import aws_sqs as sqs
+from aws_cdk import aws_s3 as s3
+
+
+class CdkStack(core.Stack):
+    def __init__(self, scope: core.Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+        stack_name = construct_id
+
+        bucket = "climatedashboard-data"
+        name = "simple-s3-ingest"
+
+        s3bucket = s3.Bucket.from_bucket_name(
+            self, f"{id}-bucket", bucket_name=bucket
```

Comment:
I'm curious how this works.  Where does `id` come from?  Should this be `construct_id`?  As written, I'm assuming it's using the [Python's builtin `id` function](https://docs.python.org/3/library/functions.html#id):

```py
In [1]: f"{id}_foo"
Out[1]: '<built-in function id>_foo'
```

Reply:
Good catch. will need to change that var name.